crates/data_components/src/flight.rs

```diff
@@ -416,6 +436,38 @@ fn query_to_stream(
     }
 }
 
+pub fn subscribe_to_append_stream(
```

Comment:
This isn't a generic Flight capability, its specific to Spice AI and should live in the Spice AI connector code.